File: string/26. Remove Duplicates from Sorted Array.py
def removeDuplicates(nums: List[int])->int:
    memo = []
    idx = len(nums)
    count = 0
    for i in range(idx):
        if nums[i] not in memo:
            nums[count] = nums[i]
            memo.append(nums[i])
            count += 1
    return count



if __name__ == "__main__":
    print("26. Remove Duplicates from Sorted Array")
    print("\nCase 1")
    s = [1,1,2]
    ans = 2
    res = removeDuplicates(s)
    print(res, "\n==> check: ", res==ans)

File: string/26. Remove_Duplicates_from_Sorted_Array.c
int removeDuplicates(int* nums, int numsSize) {
    int count = 1;
    for(int i=0; i<numsSize; i++){
        if(nums[i]!=nums[count-1]){
            nums[count] = nums[i];
            count++;
        }
    }
    return count;
}




